plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/PluginAdditionalCheckers.kt

```diff
@@ -21,7 +21,7 @@ class PluginAdditionalCheckers(session: FirSession, config: PluginConfiguration)
 
     override val declarationCheckers: DeclarationCheckers = object : DeclarationCheckers() {
         override val simpleFunctionCheckers: Set<FirSimpleFunctionChecker>
-            get() = setOf(ViperPoweredDeclarationChecker(session, config))
+            get() = setOf(ViperPoweredDeclarationChecker(session, config), UniqueDeclarationChecker(session))
```

Comment:
Please guard this via a directive, see this PR: https://github.com/jesyspa/kotlin/pull/207
You could have it imply `NEVER_CONVERT`